### ISSUE TITLE
Test the linear quadratic game with vanilla policy gradient

### ISSUE THREAD

**Issue by bchasnov** [zanedma/reinforcement_lqgame#1]
Compare solution of multi-agent vanilla policy gradient with the Nash solution of a coupled Riccati equation.

Pointers:
- modify [vpg.py](https://github.com/bchasnov/spinningup/blob/master/spinup/algos/pytorch/vpg/vpg.py) for multiple players
- see [appendix B.2](https://arxiv.org/pdf/1906.00731.pdf) for coupled Riccati equations to solve an lq game. @bchasnov has code for this already.
- compare a linear policy `u=Kx` with neural network policy `u=f(x; θ)` where `x` is state and `u` is action.
- compare the [deterministic gradient](https://arxiv.org/pdf/1801.05039.pdf) with the stochastic gradient using the log trick
